packages/blaze/attrs.js

```diff
@@ -171,8 +171,14 @@ var BooleanHandler = AttributeHandler.extend({
   }
 });
 
+var origUpdate = AttributeHandler.prototype.update;
 var ValueHandler = AttributeHandler.extend({
   update: function (element, oldValue, value) {
+    var self = this;
+    var args = arguments;
+
+    origUpdate.apply(self, args);
```

Comment:
You mean, exactly like it's [done here](https://github.com/RobertLowe/meteor/blob/phantom-needs-attributes/packages/blaze/attrs.js#L265)

Also, please keep in mind that this pull request (while it can be safely accept) is primarily calling attention to the fact that Meteor (Blaze) and PhantomJS are *not* playing nice together. Currently any application using `spiderable` and input's with bound `value` attributes will not be properly crawled/serialized. Other handlers should be checked, I do not have resources/time to commit to that.

Cheers